package.json

```diff
@@ -15,7 +16,6 @@
     "babel-brunch": "~6.0.0",
     "uglify-js-brunch": "~2.0.1"
   },
-  "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "web/static/js/phoenix.js"],
```

Comment:
@nicholaswyoung then we are subject to whatever is considered extraneous by npm. I would prefer to be explicit here, it is only a single line of code, and it will save us from accidentally including all of Phoenix one day in npm.